cogs/birthday/birthday.py

```diff
@@ -582,13 +592,33 @@ async def toggleSelfBirthday(self, ctx: Context):
             await allowSelfBirthdayConfig.set(True)
             await ctx.send(msgAllow)
 
+    def getBirthdayMessage(self, member: discord.Member) -> str:
+        """Get the birthday message.
+
+        Parameters
+        ----------
+        member: discord.Member
+            The member that we want the birthday message for.
+
+        Returns
+        -------
+        str
+            The birthday message, already formatted.
+        """
+        if self.bot.user.id == member.id:
+            return BOT_BIRTHDAY
```

Comment:
Perhaps a better name could be used? I mean, it sounds more like this constant is about the bot's birthday than the message sent on the day.
```suggestion
            return BOT_BIRTHDAY_MSG
```